packages/random-source-node/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "@aws-sdk/types": "^3.1.0",
```

Comment:
nit: standardize using `^3.2.0` as the new version is now released for JS SDK v3.
```suggestion
    "@aws-sdk/types": "^3.2.0",
```

Reply:
Yeah I think the `^` will take care of it, but yeah why not just update it to the most recent anyways

Reply:
I think the most recent version of types is 3.1.0: https://www.npmjs.com/package/@aws-sdk/types
<img width="1306" alt="Screen Shot 2021-01-12 at 11 54 53 AM" src="https://user-images.githubusercontent.com/8712303/104346132-1149a500-54cd-11eb-8c21-5d110ce18126.png">

Reply:
Yup, the `v3.2.0` was not released as there were no updates in `@aws-sdk/types`

Verified by running the command:
```
npm view @aws-sdk/types versions
```